packages/react-layouts/src/Modal/Modal.md

```diff
@@ -0,0 +1,159 @@
+The modal component provides a solution for creating dialogs, popups, lightboxes, or whatever else.
+
+```js
```

Comment:
Can we provide a simpler example before this one?

Reply:
Done